src/backend/aspen/api/views/locations.py

```diff
@@ -28,3 +40,126 @@ async def list_locations(
         response.append(LocationResponse.from_orm(row))
 
     return LocationListResponse.parse_obj({"locations": response})
+
+
+# The idea in this route is that,
+# given a country, show all divisions in that country,
```

Comment:
We also need to list the country without any divisions, in case someone wants to build country-level trees.

Reply:
added!